airflow/providers/amazon/aws/executors/ecs/Dockerfile

```diff
@@ -0,0 +1,86 @@
+# hadolint ignore=DL3007
+FROM apache/airflow:latest
+USER root
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends unzip \
+  # The below helps to keep the image size down
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
+RUN    curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+RUN    unzip awscliv2.zip && ./aws/install
+
+# Add a script to run the aws s3 sync command when the container is run
+COPY <<"EOF" /entrypoint.sh
+#!/bin/bash
+
+echo "Downloading DAGs from S3 bucket"
+aws s3 sync "$S3_URL" "$CONTAINER_DAG_PATH"
+
+exec "$@"
+EOF
+
+RUN chmod +x /entrypoint.sh
+
+USER airflow
+
+## Installing Python Dependencies
+# Python dependencies can be installed by providing a requirements.txt.
+# If the file is in a different location, use the requirements_path build argument to specify
+# the file path.
+ARG requirements_path=./requirements.txt
+ENV REQUIREMENTS_PATH=$requirements_path
```

Comment:
Should these be commented out as well, like the section below?

Reply:
I think setting these are harmless? So we chose to leave them so that the user would have to change as little as possible. CC @syedahsn

Reply:
Harmless, yes. I should have prefixed this with nit.

Reply:
This is mostly down to a preference, I wanted to reduce the number of lines that need to be commented in/out as much as possible. I'm open to either way though depending on how others feel.

Reply:
I'm going to mark this as resolved for now. Reopen if anyone else feels strongly about this one.